cnxml/__init__.py

```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
+from .validation import *
```

Comment:
Hmm... I don't agree with flake8's analysis here. It's understood, or so I thought, that it's perfectly valid to `import *` in a package's `__init__.py` if the module has defined `__all__`.